pkg/lib/registry/registry.go

```diff
@@ -399,34 +393,57 @@ func checkForBundlePaths(querier registry.GRPCQuery, bundlePaths []string) ([]st
 
 // replaces mode selects highest version as channel head and
 // prunes any bundles in the upgrade chain after the channel head.
-// check for the presence of all bundles after a replaces-mode add.
-func checkForBundles(ctx context.Context, q *sqlite.SQLQuerier, g registry.GraphLoader, required map[string]*registry.Package) error {
+// check for the presence of newly added bundles after a replaces-mode add.
+func checkForBundles(ctx context.Context, q *sqlite.SQLQuerier, g registry.GraphLoader, required []*registry.Bundle) error {
 	var errs []error
-	for _, pkg := range required {
-		graph, err := g.Generate(pkg.Name)
-		if err != nil {
-			errs = append(errs, fmt.Errorf("unable to verify added bundles for package %s: %v", pkg.Name, err))
-			continue
-		}
+checkRequired:
```

Comment:
This line and the line 404 look weird. Are these comments?

Reply:
They're go labels https://golang.org/ref/spec#Label_scopes

Reply:
I see. Not sure if they are helpful. I would prefer comments. Either way, can we correct the indentation here?